yaml/store.go

```diff
@@ -0,0 +1,180 @@
+package yaml
+
+import (
+	"fmt"
+	"github.com/autrilla/yaml"
```

Comment:
Me too, but it hasn't been reviewed yet. https://github.com/go-yaml/yaml/pull/193